docs/API/SandboxSetup.md

```diff
@@ -70,7 +70,7 @@ The chaincode console will display the message "Received REGISTERED, ready for i
 
 #### **Note on REST API port**
 
-The REST interface port is defined as port 5000 in the [core.yaml](https://github.com/hyperledger/fabric/blob/master/peer/core.yaml). If you are sending REST requests to the peer node from inside Vagrant, use port 5000. If you are sending REST requests through Swagger, the port specified in the Swagger file is port 3000. The different port emphasizes that Swagger will likely run outside of Vagrant. To send requests from the Swagger interface, set up port forwarding from host port 3000 to Vagrant port 5000 on your machine, or edit the Swagger configuration file to specify another port number of your choice.
+The REST interface port is defined as port 5000 in the [core.yaml](https://github.com/hyperledger/fabric/blob/master/peer/core.yaml). If you are sending REST requests to a peer node running in Vagrant from outside of Vagrant, also use port 5000. This mapping is defined in [Vagrantfile](https://github.com/hyperledger/fabric/blob/master/devenv/Vagrantfile)
```

Comment:
Minor nit: "peer node running in Vagrant from outside of Vagrant" seems to be missing punctuation.  I assume it was meant to say something like "peer node running in Vagrant.  From outside of Vagrant ..".  That said, I question whether we should even mention this or just leave the first sentence and drop the part of the vagrant at all.  I realize that someone that changes core.yaml would need to understand to also update the Vagrantfile, but not sure that needs/should-be mentioned here.